binaries/go-http-daemon/main.go

```diff
@@ -62,7 +63,27 @@ func executeRequest(c echo.Context) error {
 		return parseRequestError
 	}
 
-	requestResponsePair, responseErr := request.ExecuteRequest(daemonRequest.ToRequest(), daemonRequest.Profiles, daemonRequest.Variables)
+	var req request.Request
+	if daemonRequest.RequestName != "" {
+		requestOptions, err := profile.LoadRequestOptions(daemonRequest.RequestName, daemonRequest.Profiles)
```

Comment:
maybe add a method that allows you to pass a `daemonRequest`?

Reply:
That would create a circular dependency since the daemon package depends on the profile package. If I do that, I would create a dependency the other way around.
I'm still trying to figure out these names and stuff. It's very confusing where the separation is happening.

Reply:
Ah ok, sorry I didn't see that. 👍